scripts/test_env.py

```diff
@@ -18,6 +18,7 @@
 import urllib.error
 import urllib.request
 
+is_startup_benchmark = False
```

Comment:
Can you not use global variables? It worked fine without it before so I'm sure there is a way to pass this value as a function parameter